docs/source/getting_started/Installation.md

```diff
@@ -0,0 +1,47 @@
+# Installation
+Streaming Dataset installs via pip. Installing Streaming Dataset in a virtual environment is recommended to avoid any dependency conflicts. Streaming Dataset has been tested on Python 3.8, 3.9, and 3.10.
```

Comment:
Streaming Dataset here, StreamingDataset below (let's stick with class name?)

Reply:
Done as part of https://github.com/mosaicml/streaming/pull/221/commits/d70606de7f40a64dfa250aaf6b6c66c10757c6a6